app_dart/test/service/scheduler_test.dart

```diff
@@ -43,7 +44,6 @@ targets:
       - stable
     scheduler: luci
   - name: Google Internal Roll
-    postsubmit: true
```

Comment:
Any reason to remove the postsubmit|presubmit configs? it seems like that logic was not changed by this PR

Reply:
These are already the default values (if not present or not false, it will run in that environment). This is cleanup to align with how we use the ci.yaml today.